internal/kafka/internal/services/kafka.go

```diff
@@ -944,7 +945,13 @@ func (k *kafkaService) ChangeKafkaCNAMErecords(kafkaRequest *dbapi.KafkaRequest,
 		AccessKeyID:     k.awsConfig.Route53AccessKey,
 		SecretAccessKey: k.awsConfig.Route53SecretAccessKey,
 	}
-	awsClient, err := k.awsClientFactory.NewClient(awsConfig, kafkaRequest.Region)
+
+	route53Region, err := k.getRoute53RegionFromKafkaRequest(kafkaRequest)
```

Comment:
Can we remove this comment https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/656bde30a3a4d827387b25513638b38ceb1705d7/internal/kafka/internal/services/kafka.go#L943 as well?

Reply:
Removed

Reply:
Thanks @miguelsorianod